client-v2/src/main/java/com/clickhouse/client/api/ClientConfigProperties.java

```diff
@@ -104,7 +104,7 @@ public enum ClientConfigProperties {
 
     QUERY_ID("query_id"), // actually a server setting, but has client effect too
 
-    CLIENT_NETWORK_BUFFER_SIZE("client_network_buffer_size"),
+    CLIENT_NETWORK_BUFFER_SIZE("client_network_buffer_size", "8192"),

```

Comment:
lets extract it to a const

Reply:
Done -> `com.clickhouse.client.api.Client.Builder#DEFAULT_BUFFER_SIZE`